front-end/src/store/transcriptUploadSlice.js

```diff
@@ -6,15 +6,13 @@ export const transcriptUploadSlice = createSlice({
     isUploaded: false
   },
   reducers: {
-    setTranscriptUploadStatusTrue: state => {
-      state.isUploaded = true
-    },
-    setTranscriptUploadStatusFalse: state => {
-      state.isUploaded = false
+    setTranscriptUploadStatus: (state, action) => {
+      state.isUploaded = action.payload
+      //action.payload can be any object that is passed to the function
```

Comment:
I love me some comments

Reply:
Not to be contrarian, but I think this comment could be confusing. action,payload is explicitely a boolean here.